src/styles.css

```diff
@@ -26,7 +26,7 @@ header {
     font-size: 1.5rem;
 }
 
-.p-3 {
+.collapse-button {
```

Comment:
`p-3` is a class provided by bootstrap to give us padding. 
We should avoid adding extra css to stuff provided by bootstrap, and prefer our own custom classes that tell us a bit more about what the style is for.
In this case, I felt `collapse-button` was a good description of the button styling you had applied.

Reply:
The exception to that rule is, we can ABSOLUTELY override the default bootstrap button styling with our own.
The reason we can do that is because we're replacing button styles with different button styles, we're not re-purposing an existing class for something it wasn't intended to be.

Bootstrap provides `btn-primary` as a class to define primary button styles.
We could override those styles in our own CSS. Though we could just as easily define our own button in CSS with a bespoke class.